peer-prep/src/DataModel/QuestionModel.js

```diff
@@ -0,0 +1,69 @@
+// Class that stores and encapsulates the Question and its related fields
+
+class QuestionModel {
+  static QUESTION_FIELDS = {
+    ID: "Id",
+    TITLE: "Title",
+    DESCRIPTION: "Description",
+    COMPLEXITY: "Complexity",
+    CATEGORY: "Category",
+  };
```

Comment:
A good implementation to specify the properties of a question in the `QuestionModel` 👍. As a reminder, please remember that we need to consistently check all the models implemented and used across the microservices to ensure the consistency and proper functioning of our web application.

Reply:
Yes, I agree, as the Application is further developed, the Complexity would rise as well, and we need to ensure that Data Models remain consistent across Services.